File: script/make/protocol/cs/meta/BubbleProtocol.cs
using List = System.Collections.Generic.List<System.Object>;
using Map = System.Collections.Generic.Dictionary<System.String, System.Object>;

public static class BubbleProtocol
{
    public static Map GetMeta()
    {
        return new Map()
        {
            {"12101", new Map() {
                {"comment", "气泡列表"},
                {"write", new Map() { {"name", "data"}, {"type", "map"}, {"comment", ""}, {"explain", new List() {

                }}}},
                {"read", new Map() { {"name", "data"}, {"type", "list"}, {"comment", "气泡列表"}, {"explain", new List() {
                    new Map() { {"name", "data"}, {"type", "map"}, {"comment", ""}, {"explain", new List() {
                        new Map() { {"name", "bubbleId"}, {"type", "u32"}, {"comment", "气泡ID"}, {"explain", new List()} },
                        new Map() { {"name", "expireTime"}, {"type", "u32"}, {"comment", "过期时间"}, {"explain", new List()} }
                    }}}
                }}}}
            }}
        };
    }
}

File: script/make/protocol/lua/meta/BubbleProtocol.lua
return {
    [12101] = {
        ["comment"] = "气泡列表",
        ["write"] = {name = "data", type = "map", comment = "", explain = {

        }},
        ["read"] = {name = "data", type = "list", comment = "气泡列表", explain = {
            {name = "data", type = "map", comment = "", explain = {
                {name = "bubbleId", type = "u32", comment = "气泡ID", explain = {}},
                {name = "expireTime", type = "u32", comment = "过期时间", explain = {}}
            }}
        }}
    }
}